### PR TITLE
promote nginx nginx-errors opentelemetry images

### PR DESCRIPTION
**Pull request by longwuyuan**
Promoting base image for nginx, Nginx-errors & openetelemetry

Details here https://kubernetes.slack.com/archives/C021E147ZA4/p1650053036324859?thread_ts=1649867177.144969&cid=C021E147ZA4

### CODE REVIEW

File: k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml [hunk 1]
```diff
@@ -144,10 +145,12 @@
 - name: nginx-errors
   dmap:
     "sha256:5530c7fc1fa75b49e068d6b7b4f8f1c13a5834c155ba7abd5943b9057c3b925b": ["0.48.1"]
+    "sha256:06cd2dbe317505e940d0521cc69d237752800ef0b191cb4265eb0d3d2c1080e7": ["v20220415-controller-v1.2.0-beta.0-2-g81c2afd97"]
```
Comment (rikatz) [inline k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml:148]: 0.49.0 here :)

**Review by rikatz — CHANGES_REQUESTED** [review]

File: k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml (hunk 1 above)
Comment (longwuyuan) [reply k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml]: Changed to 0.49.0 on line 148 for Nginx-errors image
I was following the cloudbuild info here
image

**Review by rikatz — APPROVED** [review]
/lgtm
Tks

**Pull request merged**